spec/fizzbuzz_spec.rb

```diff
@@ -1,22 +1,31 @@
 require 'rspec'
 require 'fizzbuzz'
 
+def expect_specific(value, script)
+  lambda { |test_case| expect(script.run(test_case)).to eq value }
+end
+
 describe 'SuperFizzBuzz' do
   let(:script) { SuperFizzBuzz.new }
 
   it "returns 'Fizz' when my input is divisible by 3" do
-    expect(script.run(3)).to eq "Fizz"
+    expect_fizz = expect_specific("Fizz", script)
+
+    [3, -132, 213].each &expect_fizz
```

Comment:
Awesome! Thanks. I was wondering what the recommended way was since, although what I did is pretty general; it obfuscates the error messages. Contexts seems to be the solution there.